crates/viewer/re_view_time_series/src/view_class.rs

```diff
@@ -462,7 +463,7 @@ impl ViewClass for TimeSeriesView {
         // Get the minimum time/X value for the entire plot…
         let min_time = all_plot_series
             .iter()
-            .map(|line| line.min_time)
+            .filter_map(|line| line.points.first().map(|(t, _)| *t))
```

Comment:
so when is `min_time` the correct thing to use when it's apparently not the actual minimum time of those points? Haven't checked it's documentation, maybe that's all clear there?